web_api.py

```diff
@@ -0,0 +1,290 @@
+from datetime import datetime
+import os
+import json
+import uuid
+from typing import Dict, Union
+
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+from db.db_engine import Upload, User, UploadStatus, save_to_json, get_engine
+from flask import Flask, request, jsonify, Response, redirect, url_for, flash
+
+from explainer import OUTPUTS_FOLDER
+
+app = Flask(__name__)
+UPLOAD_FOLDER = 'uploads'
+OUTPUT_FOLDER = 'outputs'
+app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
+app.config['OUTPUT_FOLDER'] = OUTPUT_FOLDER
+
+
+@app.route('/upload', methods=['POST'])
+def upload():
```

Comment:
Separate code to small functions.  Read more: https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be